src/effects/VST3/VST3Effect.cpp

```diff
@@ -478,7 +478,7 @@ bool VST3Effect::LoadUserPreset(
    if(mEffectComponent->setState(processorState) != kResultOk)
       return false;
 
-   if(PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, wxT("ProcessorState")))
+   if(PluginSettings::HasConfigValue(*this, PluginSettings::Private, name, wxT("ControllerState")))
```

Comment:
Looks like a bug indeed

Reply:
ok thanks - I wonder if we should correct this in a separate, trivial PR so that it goes in master soon?

Reply:
Agree, @Paul-Licameli

Reply:
Agreed, write it, also eliminating the repetition of string literals.